src/components/editor/EditorReadMe.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import BaseComponent from 'lib/BaseComponent';
+
+export default class ReadMe extends BaseComponent {
```

Comment:
This is purely aesthetic for me, but I feel like "Readme" looks more natural than "ReadMe"? But that's just me being picky, if you disagree we can keep it.

Reply:
I actually had Readme at first, but then I googled it and I found that some people were using ReadMe. That, and camelcase -- but I'm happy to change it back, the word has become its own entity.

Reply:
Yeah I definitely get the logic behind ReadMe as it really does represent 2 different words, but yeah, I think as you say it's become it's own entity and it just looks weird in my eyes personally anyway.